libbpf-rs/src/util.rs

```diff
@@ -4,6 +4,12 @@ use std::path::Path;
 
 use crate::*;
 
+/// A type holding either a `L` or `R`.
+pub enum Either<L, R> {
```

Comment:
Is there a benefit to defining it here ourselves vs. using the either crate (https://docs.rs/either/1.1.0/either/enum.Either.html)?

Reply:
It's just another dependency to have publicly listed and care about, for what boils down to four lines of code. It did not seem worth it to me to pull it from the web. No other reason.